wandb/cli.py

```diff
@@ -533,10 +470,5 @@ def delete(ctx, keys):
     config.persist()
     ctx.invoke(show, changed=changed, diff=True)
 
-@config.command("run", help="Launch a job")
```

Comment:
`@config.command` is actually a sub command of the `@config` group.  Moved it up in the file and used `@cli.command` which is the top level group.